include/psa/crypto_extra.h

```diff
@@ -116,6 +116,43 @@ static inline psa_algorithm_t psa_get_key_enrollment_algorithm(
  */
 void mbedtls_psa_crypto_free( void );
 
+/** \brief Statistics about
+ * resource consumption related to the PSA keystore.
```

Comment:
Why is this broken in two lines in such a strange manner?

Reply:
Probably the result of having edited an earlier draft. Does it matter?